cmd/gogio/androidbuild.go

```diff
@@ -569,6 +593,94 @@ func signAPK(tmpDir string, tools *androidTools, bi *buildInfo) error {
 	return nil
 }
 
+func signAAB(tmpDir string, tools *androidTools, bi *buildInfo) error {
+	aabFile := *destPath
+	if aabFile == "" {
+		dir, _ := os.Getwd()
+		aabFile = filepath.Join(dir, fmt.Sprintf("%s.aab", bi.name))
+	}
+
+	if filepath.Ext(aabFile) != ".aab" {
+		return fmt.Errorf("the specified output %q does not end in '.aab'", aabFile)
+	}
+
+	allBundleTools, err := filepath.Glob(filepath.Join(tools.buildtools, "bundletool*.jar"))
+	if err != nil {
+		return err
+	}
+
+	bundletool := ""
+	for _, v := range allBundleTools {
+		bundletool = v
+		break
+	}
+
+	if bundletool == "" {
+		return fmt.Errorf("bundletool was not found at %s. Download it from https://github.com/google/bundletool/releases and move to the respective folder.", tools.buildtools)
+	}
+
+	_, err = runCmd(exec.Command(
+		"java",
+		"-jar", bundletool,
+		"build-bundle",
+		"--modules="+filepath.Join(tmpDir, "app.zip"),
+		"--output="+filepath.Join(tmpDir, "app.aa_"),
+	))
+	if err != nil {
+		return err
+	}
+
+	_, err = runCmd(exec.Command(
+		filepath.Join(tools.buildtools, "zipalign"),
+		"-f",
+		"4", // 32-bit alignment.
+		filepath.Join(tmpDir, "app.aa_"),
+		aabFile,
+	))
+	if err != nil {
+		return err
+	}
+
+	if bi.key == "" {
+		if err := defaultAndroidKeystore(tmpDir, bi); err != nil {
+			return err
+		}
+	}
+
+	result, err := runCmd(exec.Command(
+		"keytool",
+		"-keystore", bi.key,
+		"-list",
+		"-keypass", bi.password,
+		"-v",
+	))
+	if err != nil {
+		return err
+	}
+
+	matchAlias, err := regexp.Compile("Alias name:(.*)")
```

Comment:
This doesn't seem worth using the regexp package for. I suggest fmt.Sscanf or just strings.HasPrefix and slicing.